src/base/server_type.cpp

```diff
@@ -73,8 +81,35 @@ void init_page_server_hosts ()
 	      hosts.c_str ());
       return;
     }
+  g_pageserver_port = port;
 
   // host and port seem to be OK
-  std::string host = hosts.substr (0, col_pos);
-  er_log_debug (ARG_FILE_LINE, "Page server hosts: %s port: %d\n", host.c_str (), port);
+  g_pageserver_hostname = hosts.substr (0, col_pos);
+  er_log_debug (ARG_FILE_LINE, "Page server hosts: %s port: %d\n", g_pageserver_hostname.c_str (), g_pageserver_port);
+
+  connect_to_pageserver (g_pageserver_hostname, g_pageserver_port, db_name);
+}
+
+void connect_to_pageserver (std::string host, int port, const char* db_name)
+{
+  assert (get_server_type () == SERVER_TYPE_TRANSACTION);
+
+  // connect to page server
+  cubcomm::server_channel srv_chn (db_name);
+
+  srv_chn.set_channel_name ("TODO");
+  srv_chn.set_debug_dump_data (true); // TODO: is_debug_communication_data_dump_enabled ());
```

Comment:
We discussed to remove debug dump data for now.

Reply:
done